Sources/Inject/Integrations/Hosts.swift

```diff
@@ -144,5 +151,15 @@ public class _InjectableViewHost<Hosted: InjectViewType>: InjectViewType {
         instance[keyPath: keyPath]
     }
 }
+#else
+
+extension Inject {
+    public static func ViewControllerHost(_ viewController: InjectViewControllerType) -> InjectViewControllerType {
```

Comment:
The thing with this approach is that it drops the idea of `@autoclosure` initialization, so it won't really restore fresh state for your hosted content?

Reply:
It can be replaced with `@autoclosure`, but I'm not sure what's the use case for that in the release code? 

In release code I assume we just want to return the argument without any additional operations, similarly to SwiftUI where we return self, right?
```
func enableInjection() -> Self { self }
```

Let's say I've got in my code:
```
rootViewController.pushViewController(builder.build(), animated: true)
// after migration
rootViewController.pushViewController(Inject.ViewControllerHost(builder.build()), animated: true)
```
When I migrate it to Inject I expect it to use `autoclosure` to keep it fresh when hot-reload happens, but in release code I just expect it to use the instance returned from `builder.build()` without any additional operations.

Reply:
btw. I think it would be also good to add to README this case to common pitfalls:
```
let viewController = builder.build()
Inject.ViewControllerHost(viewController)
```

I did that the first time I used Inject, because it was not clear from the outside that it relies on `autoclosure` :D

Reply:
oh I think I misread that when I made the comment (sorry just woke up 😂), your argumentation makes sense to me!

Do you want to add a line to readme about this? I'll merge after

Reply:
I'll do that :)

Reply:
@krzysztofzablocki done :)